yaksh/views.py

```diff
@@ -594,11 +597,17 @@ def check(request, q_id, attempt_num=None, questionpaper_id=None):
         else:
             user_answer = request.POST.get('answer')
         if not user_answer:
-            msg = ["Please submit a valid option or code"]
+            msg = "Please submit a valid option or code"
```

Comment:
Why is this option or code?  Isn't it just an option?

Reply:
Or maybe option/value?

Reply:
This message is displayed for both code and non-code questions if user answer is empty. So the message is common for both.